tests/kernel/mem_heap/mheap_api_concept/src/test_mheap_concept.c

```diff
@@ -81,24 +81,44 @@ void test_mheap_block_desc(void)
 	 * TESTPOINT: The kernel uses the first 16 bytes of any memory block
 	 * allocated from the heap memory pool to save the block descriptor
 	 * information it needs to later free the block. Consequently, an
-	 * application’s request for an N byte chunk of heap memory requires a
-	 * block that is at least (N+16) bytes long.
-	 * Test steps:
-	 * initial memory heap status (F for free, U for used):
-	 *    64F, 64F, 64F, 64F
-	 * 1. request 4 blocks: each (64-16) bytes, indeed 64-byte allocated
-	 * 2. verify no more free blocks, any further allocation failed
-	 */
-	for (int i = 0; i < BLK_NUM_MAX; i++) {
-		block[i] = k_malloc(BLK_SIZE_EXCLUDE_DESC);
-		zassert_not_null(block[i], NULL);
-	}
-	/* verify no more free blocks available*/
-	block_fail = k_malloc(BLK_SIZE_MIN);
-	zassert_is_null(block_fail, NULL);
+	 * application’s request for an N byte chunk of heap memory requires
```

Comment:
It's subtle -- there's a UTF-8 printer's quote I changed to a regular quote: "application's"